docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,138 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Design Tokens Taxonomy" />
+
+# 🚧 WIP
+
+## Design system namespace
+
+All tokens start with the namespace for the design system, `cpd`.
+
+## Token tiers
+
+Our design tokens are conceptually divided into three tiers:
+
+- System
+- Reference
+- Components
+
+The top level is global and not aware of context. Subsequent tiers are context-aware, each one being more precise.
+
+System tokens describe plain values, while reference tokens describe their purpose in specific contexts. Component tokens describe the intended use in the context of a specific component.
+
+With three kinds of tokens, we can update design decisions globally or apply a change to a single component.
+
+### System
+
+Namespace: `sys`
+🔴 TODO: Consider removing namespace.
+
+Use for global, immutable literal values.
+
+### Reference
+
+Namespace: `ref`
+🔴 TODO: Consider removing namespace.
```

Comment:
Nothing currently lets us define the namespace from the token structure

Reply:
Already discussed here https://github.com/vector-im/compound/pull/7#discussion_r1092160914